config/initializers/add_uploads_from_csv.rb

```diff
@@ -26,7 +26,9 @@ def perform(csv_data, exhibit, _user)
   # Given a filepath, ensure the file exists and make a Spotlight::FeaturedImage object
   def fetch_image_from_local_disk(file)
     image = Spotlight::FeaturedImage.new
-    image.image.store!(File.open(file))
+    import_dir = ENV['IMPORT_DIR'] || ''
+    filepath = File.join(import_dir, file)
```

Comment:
If `file` is an absolute path (i.e. starting with `/`) then `File.join` will throw away the first parameter.  That means we can use absolute paths in the CSV, but we should also document it so anyone preparing the metadata doesn't put a slash at the beginning of paths without meaning to.

Reply:
See the README. Does that look like enough documentation?